pkgs/os-specific/linux/iputils/default.nix

```diff
@@ -35,20 +35,21 @@ stdenv.mkDerivation rec {
   installPhase =
     ''
       mkdir -p $out/bin
-      cp -p ping tracepath clockdiff arping rdisc $out/bin/
+      cp -p ping tracepath clockdiff arping rdisc rarpd ninfod/ninfod $out/bin/
```

Comment:
The 3 lines below should be deleted in this case, because `ninfod` is added here.

Reply:
Fixed by removing the unconditional copy (this was a bit I got wrong while rebasing)